apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,96 @@
+import defaultAvatar from "~/assets/img/default-avatar.png";
+import { Button, Loader } from "~/libs/components/components.js";
+import { DataStatus } from "~/libs/enums/enums.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useCallback,
+	useEffect,
+	useParams,
+	useState,
+} from "~/libs/hooks/hooks.js";
+import {
+	actions as usersActions,
+	type UserUpdateRequestDto,
+} from "~/modules/users/users.js";
+
+import { UpdateUserForm } from "./libs/components/components.js";
+import styles from "./styles.module.css";
+
+const Profile: React.FC = () => {
+	const { id } = useParams();
+	const dispatch = useAppDispatch();
+	const { dataStatus, user } = useAppSelector(({ users }) => ({
+		dataStatus: users.dataStatus,
+		user: users.user,
+	}));
+	const [isUpdating, setIsUpdating] = useState<boolean>(false);
+
+	useEffect(() => {
+		if (id) {
+			void dispatch(usersActions.getById({ id: +id }));
+		}
+	}, [dispatch, id]);
+
+	useEffect(() => {
+		if (dataStatus === DataStatus.FULFILLED) {
+			setIsUpdating(false);
+		}
+	}, [dataStatus]);
+
+	const toggleIsUpdate = useCallback((): void => {
+		setIsUpdating(!isUpdating);
+	}, [setIsUpdating, isUpdating]);
+
+	const handleUpdateSubmit = useCallback(
+		(payload: UserUpdateRequestDto): void => {
+			if (id) {
+				void dispatch(usersActions.update({ data: payload, id: +id }));
+			}
```

Comment:
Looks like anyone can update any profile just by changing id in the url. Let's make the route `/profile` and we will display only the current user data